markdown/explore-dmr/explore-dmr.md

```diff
@@ -0,0 +1,295 @@
+    Finally click "Create Link and Test Connection"
+    ![Configure Connection Parameters](img/configureConnectionParameters.png)
+
+11. Wait for Link and Test:
+    The broker will now attempt to establish a link with the specified DMR Link mate.  Then once the connection is negotiated the brokers will run bi-directional communication tests.
+    ![Wait for Link and Test](img/waitForLinkAndTest.png)
+
+12. Successful DMR Link:
+    After communication tests we will have a successful DMR Link.  
+    <br>⚠️ If you encounter an error please review a DMR troubleshooting steps to determine how to correct the error and build a successful DMR Link between brokers
+    ![Successful DMR Link](img/successfulLink.png)
+
+**❗️❗️♻️ At this point if you would like to include more than 2 brokers in your Event Mesh repeat steps 6 to 12 for each additional broker you wish to include in the mesh.  <br>⚠️Note that events will not traverse N brokers to reach a desired broker so you have to form a direct DMR Link between each and every broker for form a full Event Mesh**
+
+### Conclusion
+In the prevous steps we have provisioned a cluster on each broker to prepare them to participate in an Event Mesh.  Then we created a DMR Link between the brokers to form the Event Mesh.  Events will no be intelligently and Dynamically Routed from the Solace PubSub+ brokers where they are produced to the broker where there are interested subscribers. Continue to learn how we can explore the behavior of events over a DMR Link. 
+
+
+
+
+## Exploration DMR with “Try Me” tab
+Duration: 0:02:00
+
+The Solace Try Me Tab is small CodePen application which was created to provide a simple way of publishing and subscribing to Solace topics and queues.  The application produces and consumes WebSocket (WS) events to Solace PubSub+ brokers over a network.  Thus is it not required that the Try Me tool be used to connect to a local broker, however the tooling is setup to make connecting to the "local" Solace Broker effortless. Using the Try Me tool you can explore the following Solace features:
+* Publish Direct Messages
+* Publish Persistent Messages
+* Publish to topics
+* Publish directly to queues
+* Subscribe to Solace topics
+* Subscribe to Solace queues
+* Subscribe to Solace topics using Solace wild cards
+* Experience Solace Broker translate different message protocols (requires the use of Try Me and an additional publisher or subscriber).
+
+<br>
+
+**In the next section we will explore how using the Try Me tool to Publish and Subscribe only to a single individual Solace Broker will allow us to obtain events from any accessible topic published to any of the Solace Brokers connected to our Solace DMR Event Mesh.** 
+<br>🆘!!!Maybe here I should include some topic architecture for 2 brokers that will allow you to see the DMR Event Mesh in action?
+
+## Setting up and running "Try Me"
+Duration 00:10:00
```

Comment:
I believe this should be `Duration 0:10:00` you have an extra zero at the beginning which broke errythang